test_rmw_implementation/test/test_init_shutdown.cpp

```diff
@@ -0,0 +1,46 @@
+// Copyright 2020 Open Source Robotics Foundation, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#include <gtest/gtest.h>
+
+#include "osrf_testing_tools_cpp/scope_exit.hpp"
+
+#include "rcutils/allocator.h"
+#include "rcutils/error_handling.h"
+
+#include "rmw/rmw.h"
+
+
+TEST(TestInitShutdown, init_shutdown) {
```

Comment:
Do these test class names need to be modified for their rmw_implementation?
``
TEST(CLASSNAME(TestInitShutdown, RMW_IMPLEMENTATION), init_shutdown)
```

Reply:
Yes! Good catch!

Reply:
Was just typing this haha

Reply:
See https://github.com/ros2/rmw_implementation/pull/106/commits/228f34bbbffba532cc91a5edba2b20d107e4987c.